test/infrastructure/docker/examples/machine-pool.yaml

```
@@ -35,7 +35,7 @@ metadata:
   namespace: default
 spec:
   replicas: 1
-  version: v1.32.0-rc.1
+  version: v1.32.0.1
```

```suggestion
  version: v1.32.0
```
Same in other places